api/CanvasRenderingContext2D.json

```diff
@@ -2125,15 +2125,21 @@
                 "prefix": "webkit"
               }
             ],
-            "edge": {
-              "version_added": "15"
-            },
+            "edge": [
+              {
+                "version_added": "15"
+              },
+              {
+                "version_added": "12",
+                "prefix": "ms"
```

Comment:
Can you link the test used for this? Also I assume it was removed in 79 or earlier?